_pytest/test_unwrap_attachments.py

```diff
@@ -28,6 +28,21 @@
             "Attachment text",
         ]),
     },
+    {
+        'input_message': {'attachments': [{
+            'title': 'Title',
+            'text': 'Attachment text',
+            'title_link': 'http://title.link',
+            'image_url': 'http://image.url',
+            'fallback': 'Fallback',
+        }]},
+        'input_text_before': "",
+        'output': "\n".join([
+            "Title (http://title.link)",
+            "http://image.url",
+            "Attachment text",
```

Comment:
I think it'd look better if the text came before the image link rather than after.

Otherwise, lgtm! :+1: